v2/containeranalysis/client.go

```diff
@@ -188,15 +189,12 @@ func (g *Client) GetBuildDetail(ctx context.Context, ref reference.Canonical) (r
 
 // NewClient creates a new containeranalysis Grafeas Client.
 func NewClient(ctx context.Context, binauthProject string, keyring signer.AttestationSigner) (*Client, error) {
-	var err error
-
-	caClient, err := containeranalysisapi.NewClient(ctx)
+	grafeasClient, err := grafeasv1.NewClient(ctx, option.WithEndpoint("containeranalysis.googleapis.com:443"), option.WithScopes(containeranalysisapi.DefaultAuthScopes()...))
```

Comment:
These values are rather magical.

My goal was to inline [lines 97-100](https://github.com/googleapis/google-cloud-go/blob/50ca760f8a4415a78de2556fc545e340eae98715/containeranalysis/apiv1/container_analysis_client.go#L97-L100) of `containeranalysisapi.NewClient`, so these are the [defaultClientOptions()](https://github.com/googleapis/google-cloud-go/blob/50ca760f8a4415a78de2556fc545e340eae98715/containeranalysis/apiv1/container_analysis_client.go#L41-L46), which unfortunately isn't exported.

Reply:
thanks for the context! Can we add this as a comment?